augur/validate.py

```diff
@@ -59,18 +66,108 @@ def load_json(path):
     return jsonToValidate
 
 def validate_json(jsonToValidate, schema, filename):
+    # See <https://python-jsonschema.readthedocs.io/en/v3.2.0/errors/> and
```

Comment:
Some words dropped out at the end of this sentence in 59b5505d91f3774fada153cbb4c9bdeeb595645e:

> Previously only the
top-level error was reported and it was worse than useless as it was
incredibly long and contained no actionable .

Reply:
Whoops. Fixed in 750db2b3.